distribution/ltp/lite/metadata

```diff
@@ -8,7 +8,7 @@ destructive=no
 
 [restraint]
 entry_point=make run
-max_time=300m
-dependencies=automake;autoconf;procmail;flex;bison;rsyslog;util-linux;rpm-build;gcc;wget;kernel-headers;bc;kernel-devel;libaio-devel;libcap-devel;libcgroup;strace
-softDependencies=numactl;numactl-devel;redhat-lsb;ntpdate
+max_time=180m
```

Comment:
The shorter timed caused LTP to be killed by localwatchdog way too often on slower / busier machines, hence it was changed to the larger value. I believe it would be better to keep it